backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java

```diff
@@ -374,19 +329,13 @@ private void deleteDatasets(Session session) {
             datasetIds, ModelDBServiceResourceTypes.DATASET);
         var transaction = session.beginTransaction();
         var updateDeletedStatusDatasetVersionQueryString =
-            new StringBuilder("UPDATE ")
-                .append(DatasetVersionEntity.class.getSimpleName())
-                .append(" dv ")
-                .append("SET dv.")
-                .append(ModelDBConstants.DELETED)
-                .append(" = :deleted ")
-                .append(" WHERE dv.")
+            new StringBuilder("UPDATE DatasetVersionEntity dv SET dv.deleted = :deleted WHERE dv.")
```

Comment:
same as above

Reply:
Updated to string instead of string builder